packages/react-menu/src/components/Menu/useMenu.tsx

```diff
@@ -38,6 +41,8 @@ export const useMenu = (props: MenuProps, ref: React.Ref<HTMLElement>, defaultPr
     console.warn('Menu can only take one MenuTrigger and one MenuList as children');
   }
 
+  const { targetRef: triggerRef, containerRef } = usePopper({ align: state.align, position: state.position });
```

Comment:
```suggestion
  const { targetRef, containerRef } = usePopper({ align: state.align, position: state.position });
```

_nit_

Reply:
Oops, sorry, it was too late for me. I have seen it as `targetRef: targetRef` 🤦‍♂️